docs/learn/overview/how-it-works/overview.md

```diff
@@ -1 +1,122 @@
+- **Maintain Compliance and Performance:** Nodes must run the approved software stack and meet hardware requirements, including the use of AMD SEV-SNP technology. Operators are expected to maintain high uptime and performance.
+- **Follow Protocol Rules:** Operators must adhere to the protocol's consensus mechanisms and operational guidelines.
+
+### Node Operator List
+
+Current node operators include:
+
+- **Lit Protocol (our node)**
+- [Hypha](https://hypha.coop/)
+- [Thunderhead](https://thunderhead.xyz/)
+- [Terminal3](https://www.terminal3.io/)
+- [Imperator](https://www.imperator.co/)
+- [01node](https://01node.com/)
+- [Cheqd](https://cheqd.io/)
+
+Operators joining after launch:
+
+- [HireNodes](https://hirenodes.io/)
+- [ETHGlobal](https://ethglobal.com/)
+- [Zerion](https://zerion.io/)
+
+### The Lit Protocol Token
+
+The Lit Protocol Token is the native utility token for the network and serves multiple functions:
+
+- **Staking:** Node operators stake the token to secure their participation and align economic incentives.
+- **Incentives and Rewards:** Operators receive the token as compensation for providing services and maintaining the network.
+- **Payment for Services:** Developers use the token to pay for network services, such as executing Lit Actions, decryption operations, and signing data with [Programmable Key Pairs (PKPs)](../../signing-data/pkps.md).
+
+:::info
+
+**Note:** The Lit Protocol token **is not yet live**. Currently, a test token (tstLPX) is used during the Mainnet Beta phase. The official token will be released when the v1 network launches later this year. Subscribe to updates [here](https://spark.litprotocol.com/).
+
+Developers can claim test tokens from the [verified faucet](https://chronicle-yellowstone-faucet.getlit.dev/).
+
+:::
+
+## How Lit Protocol Enables Secure and Private Operations
+
+### Secure Key Management Without Custodians
+
+Lit Protocol provides a decentralized alternative to traditional key management solutions, which often rely on centralized custodians or require users to manage their own private keys.
+
+- **Non-Custodial Key Generation:** Users can have keys generated and managed securely without any single entity controlling the full private key.
+- **Flexible Authentication Methods:** Lit allows for key generation using traditional Web2 authentication methods like social OAuth (e.g. Google, Facebook, X (Twitter)). This enables seamless user onboarding by leveraging familiar login mechanisms while maintaining decentralized key management.
+- **Threshold Signatures:** Operations like signing and decryption require collaboration among multiple nodes, enhancing security.
+- **Access Control Policies:** Developers can define complex [access control conditions](../../encryption-access-control/overview.md) based on various conditions, enabling fine-grained control over who can access certain data or perform specific actions.
+
+### Confidential Computing with TEEs
```

Comment:
I feel like this section is very similar to the TEE section earlier in this page